packages/settings-library/src/settings_library/postgres.py

```diff
@@ -31,7 +31,9 @@ class PostgresSettings(BaseCustomSettings):
     )
 
     POSTGRES_CLIENT_NAME: Optional[str] = Field(
-        None, description="Name of the application connecting the postgres database"
+        None,
+        description="Name of the application connecting the postgres database",
+        env=["HOST", "POSTGRES_CLIENT_NAME"],
```

Comment:
In the case of HOST, please doc the rationale behind this or an example , i.e. something like ``$HOST`` is typically set in the service as ``--hostname="{{.Node.Hostname}}-{{.Node.ID}}-{{.Service.Name}} ``